cleanlab/datalab/internal/issue_finder.py

```diff
@@ -70,8 +70,9 @@ class IssueFinder:
     `Datalab.find_issues` method which internally utilizes an IssueFinder instance.
     """
 
-    def __init__(self, datalab: "Datalab", verbosity=1):
+    def __init__(self, datalab: "Datalab", task: str = "classification", verbosity=1):
```

Comment:
```suggestion
    def __init__(self, datalab: "Datalab", task: str, verbosity=1):
```

Reply:
I think `IssueFinder` doesn't have to directly passed `task` as it could be extracted from `Datalab`. 
Btw. in few other places removal of default value of classification causes failure of test (missing value for positional argument `task`), I've updated those tests.  `ImageLab` was also updated.